tests/e2e/assistant.spec.js

```diff
@@ -29,8 +38,1269 @@ const MediaServices = {
+        "Religious, Ethical and Cultural",
+        "Fairness, Equality and Rights",
+        "Security, Defense and Well-being",
+        "Politics",
+      ],
+      genre: ["Satire"], // I don't think so
+      persuasionTechniques: [
+        // I'm not sure about any of these, tbh...
+        "Doubt",
+        "Loaded language",
+        "Name calling or labeling",
+        "Repetition",
+        "Slogans",
+      ],
+      subjectivity: ["Subjective sentences detected (1/9)"]
+    }
+  },
+  {
+    url: "https://www.instagram.com/reel/CvstPHSg6Yd/?utm_source=ig_web_copy_link&igshid=MzRlODBiNWFlZA==",
+    mediaType: MediaType.video,
+    services: [MediaServices.videoDownloadGeneric],
+    namedEntities: {
+      Person: [
+        "Nicky Youre",
+      ],
+      Hashtag: [
+        "#reeloftheweek",
+      ],
+      UserID: [
+        "louietheraccoon",
+      ]
+    },
+    extractedURLAnalyses: {
+      // Fails
+    },
+    credibilitySignals: {
+      topic: [
+        "Religious, Ethical and Cultural",
+        "Politics",
+      ],
+      genre: ["Satire"], // I don't think so
+      persuasionTechniques: [
+        "Repetition", // No
+      ],
+      subjectivity: ["None detected"]
+    }
+  },
+  // Twitter
+  {
+    url: "https://twitter.com/NatGeo/status/1285094685485289472",
+    mediaType: MediaType.none,
+    namedEntities: {
+      Organization: [
+        // "sun", // Should be a location, if anything?
+      ],
+      URL: [
+        // Another one of those undefined URLs
+      ]
+    },
+    extractedURLAnalyses: {
+      "https://on.natgeo.com/3eOgJRg": LinkResult.none
+    },
+    credibilitySignals: {
+      topic: [
+        "Security, Defense and Well-being",
+      ],
+      genre: ["Satire"], // I don't think so
+      persuasionTechniques: [
+        "Appeal to fear/prejudice",
+        "Repetition", // No
+      ],
+      subjectivity: ["None detected"]
+    }
+  },
+  {
+    url: "https://twitter.com/ProfMarkMaslin/status/1679016022190313473",
+    mediaType: MediaType.none,
+    namedEntities: {
+      Organization: [
+        // "Global", // No
+      ],
+      URL: [
+        // Another one of those undefined URLs
+      ]
+    },
+    extractedURLAnalyses: {
+      // Fails
+    },
+    credibilitySignals: {
+      topic: [
+        "Economy and Resources",
+      ],
+      genre: ["Opinionated News"], // I don't think so
+      persuasionTechniques: [
+        "Appeal to fear/prejudice",
+        "Exaggeration or minimisation",
+        "Loaded language",
+        "Repetition",
+      ],
+      subjectivity: ["None detected"]
+    }
+  },
+  // Mastodon
+  {
+    url: "https://fosstodon.org/@davidwilby/109313349220686853",
+    mediaType: MediaType.none,
+    credibilitySignals: {
+      topic: [
+        // These are not correct
+        "Security, Defense and Well-being",
+        "Health and Safety"
+      ],
+      genre: ["Satire"], // I don't think so (Seems to be the default when it doesn't know?)
```

Comment:
@ianroberts The genre classifier has three classes: subjective reporting, objective reporting and satire. Does it always select one? Or is there ever "No genre detected" for example

Reply:
@LesyaR can you answer this one?

Reply:
@ianroberts @rosannamilner Thank you for the question. The model is trained in such a way that there is always one class (with the highest probability) returned. This corresponds to how the dataset the model was trained on is annotated - there is exactly one gold-standard class per input.

Reply:
Thanks @LesyaR. @jmafoster1 There isn't something to change here so this ready to go